### ISSUE TITLE
Playtesting - Thoughts on the Janitor Stall

### ISSUE THREAD

**Issue by Nextej** [loiste-interactive/Obenseuer-Issues#2727]
I personally don't feel like a whole stall is appropriate for a janitor to have. Like he just sits there on cold for 8 hours, offering nothing but an off-hour service for tenements, despite the fact that new tenements don't pop up out of nowhere for him to look for new clients, so he logically would already be acquainted with/working for all the tenement administrations in the district.

Feels that if he was offering services, he either would be dropping business cards into the mailboxes with a number to call on, putting up advertisement posters or be met in person while he is working in Tenement A or Kolhola. Or all above.

**There's also some alternatives worth to consider in addition to the Janitor persona, perhaps available on different stages of the story/tenement progression:**

- Bunch of hardy boys sitting at the pub by the evenings looking for some cash, fitting idea to employ them for daily (or other custom durations) water resupplies. Perhaps they would be able to provide both clean water (once district is repaired) or a shroomwater, to player's choice.

- Malone could act as the entry water supplier, getting his lazy ass to the nearby canal, low capacity (like 2 sinks a day max). Unless whole Malone's helping idea is abandoned.

- Janitor could offer a steady supply of clean water, but also perhaps cleaning out bucket toilets, regular toilets (if there was a mechanic of clogging/staining), clear out trash bins all over the tenement. As stated above, after player is in position of having any services served by him, he'll start dropping in the mail the businesses cards with a number to call or you'll begin to be able to rarely meet him around Kolhola and Tenement A.

- Wholeseller warehouse in central district zone that will be able to resupply you daily in bulk with Spring Water and vending machines resupply, only by visiting them physically.

In short, I don't think a physical stall fits both the profession and the setting for the janitor to have, nor I think they should be the starting choice for the player.

**Comment by Nextej** [comment]
Regarding the idea that Janitor services should be callable on a phone from a business card found in a mail. Some concept for how it would look, the phone number should obviously be transcribed into item's description.

<img width="468" height="340" alt="Image" src="https://github.com/user-attachments/assets/9004007b-fd2c-4e18-9482-2f0b99cb9280" />

<img width="440" height="256" alt="Image" src="https://github.com/user-attachments/assets/0c6ee1e6-b84f-4f9b-b27f-9a18f50027c5" style="image-rendering: pixelated;"/>

Other services could also get their business cards you get randomly in mail, like Rob-In or Lawyer (in both cases telling you to visit bazaar to receive service when called).
